homeassistant/components/simplisafe/__init__.py

```diff
@@ -597,6 +597,14 @@ def __init__(self, simplisafe, system, name, *, serial=None):
             ATTR_SYSTEM_ID: system.system_id,
         }
 
+        self._device_info = {
+            "identifiers": {(DOMAIN, self._system.system_id)},
```

Comment:
the lines above this use system param directly rather than the stored attribute prob best to be consistent